specification/ingest/_types/Processors.ts

```diff
@@ -147,6 +147,13 @@ export class ProcessorContainer {
    * @doc_id rename-processor
    */
   rename?: RenameProcessor
+  /**
+   * Routes a document to another target index or data stream.
+   * When setting the `destination` option, the target is explicitly specified and the dataset and namespace options can’t be set.
+   * When the `destination` option is not set, this processor is in a data stream mode. Note that in this mode, the reroute processor can only be used on data streams that follow the data stream naming scheme.
+   * @doc_id reroute-processor
```

Comment:
For this to work you need to add a line in `specification/_docs_ids/table.csv` with the link to the documentation.

Reply:
Ahh I see

Added